parsers/CL_SING.py

```diff
@@ -85,7 +86,7 @@ def data_processer(data):
 
         for key in datapoint.keys():
             if not key in plant_map.keys():
-                print '{} is missing from the CL_SING plant mapping.'.format(key)
+                print('{} is missing from the CL_SING plant mapping.'.format(key))
 
         mapped_plants = [(plant_map.get(plant, 'unknown'), val) for plant, val in datapoint.iteritems()]
```

Comment:
Doesn't all `iteritems()` calls need to be changed to `items()`?

Reply:
Where is circle.yml did you add the pyenv stuff?

On iteritems(), Yes.  This is the kinda stuff that flake8 would catch if it was running on Python 3.